_source/_posts/2020-01-21-angular-material-login.md

```diff
@@ -37,70 +39,67 @@ In this tutorial, I'll show you how to create a login form like the one above. Y
 
 ## Build an Angular Material App with Secure Login
 
-To get started, you will need to install the Angular command-line tool. I will assume that you have some familiarity with JavaScript and that you have Node installed on your system together with the Node Package Manager `npm`. Open a terminal and type the following command.
+To get started, you will need the Angular command-line tool. I will assume that you have some familiarity with JavaScript and that you have Node installed on your system together with the Node Package Manager `npm`.
+
+Open a terminal and run the following command to verify if you have Angular CLI installed.
+
+```bash
+ng version
+```
+
+If you see output with Angular CLI version installed, you're good to move on to the next step. Otherwise, you don't already have Angular CLI installed, so install it by running.
 
 ```bash
-npm install -g @angular/cli@11
+npm install -g @angular/cli
 ```
 
-Depending on your operating system, you might have to run this using the `sudo` command. This will install the Angular command-line tool on your system. It lets you use the `ng` command to set up and manipulate Angular applications. To create a new application navigate to a directory of your choice and run the following.
+Depending on your operating system, you might have to run this using the `sudo` command. This will install the Angular command-line tool on your system. It lets you use the `ng` command to set up and manipulate Angular applications. In order to create an Angular app for the version we want, we'll use `npx` to create a new app. To create a new application navigate to a directory of your choice and run the following.
 
 ```bash
-ng new material-tic-tac-toe
+npx @angular/cli@13 new material-tic-tac-toe
 ```
 
-You will be asked three questions. Answer **Yes** to the first two questions. This will enable stricter type checking and set up the router in your application. The router is responsible for letting the user navigate between different parts of the app and updating the browser's URL without actually reloading the page. For the third question, accept the default choice **CSS**. This application will use simple CSS stylesheets, but you can see how easy it is to switch to a different stylesheet technology with Angular.
+You will be asked two questions. Answer **Yes** to the first question. This will set up the router in your application. The router is responsible for letting the user navigate between different parts of the app and updating the browser's URL without actually reloading the page. For the third question, accept the default choice **CSS**. This application will use simple CSS stylesheets, but you can see how easy it is to switch to a different stylesheet technology with Angular.
```

Comment:
```suggestion
You will be asked two questions. Answer **Yes** to the first question. This will set up the router in your application. The router is responsible for letting the user navigate between different parts of the app and updating the browser's URL without actually reloading the page. For the second question, accept the default choice **CSS**. This application will use simple CSS stylesheets, but you can see how easy it is to switch to a different stylesheet technology with Angular.
```